scania-digital-design-system/sdds
# fix(radio button): removed default slot

**Describe pull-request**  
Removed default slot and added named label slot.

**Solving issue**  
Fixes: -

**How to test**  
1. Go to storyook link below
2. Check in Components -> Radio button -> Web Component
3. Check that the label works as intended.A

**Suggested test steps**
